e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-enabled/gef-deal-add-bank-review-date.spec.js

```diff
@@ -0,0 +1,147 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../../../e2e-fixtures';
+import { MOCK_FACILITY_ONE } from '../../../../../fixtures/mock-gef-facilities';
+import { MOCK_APPLICATION_AIN } from '../../../../../fixtures/mock-gef-deals';
+import { DEAL_TYPE } from '../../../../../../../gef/cypress/fixtures/constants';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import dateConstants from '../../../../../../../e2e-fixtures/dateConstants';
+
+if (Cypress.env('FF_TFM_FACILITY_END_DATE_ENABLED') === 'true') {
+  context('Amendments - GEF deal add bank review date', () => {
+    let dealId;
+    let facility;
+
+    const MOCK_GEF_FACILITY = {
+      ...MOCK_FACILITY_ONE,
+      hasBeenIssued: true,
+      isUsingFacilityEndDate: undefined,
+      facilityEndDate: undefined,
+      bankReviewDate: undefined,
+    };
+
+    before(() => {
+      // inserts a gef deal
+      cy.insertOneGefDeal(MOCK_APPLICATION_AIN, BANK1_MAKER1).then((insertedDeal) => {
+        dealId = insertedDeal._id;
+        // updates a gef deal so has relevant fields
+        cy.updateGefDeal(dealId, MOCK_APPLICATION_AIN, BANK1_MAKER1);
+
+        cy.createGefFacilities(dealId, [MOCK_GEF_FACILITY], BANK1_MAKER1).then((createdFacility) => {
+          facility = createdFacility.details;
+        });
+
+        cy.submitDeal(dealId, DEAL_TYPE.GEF, T1_USER_1);
+      });
+    });
+
+    beforeEach(() => {
+      cy.login(PIM_USER_1);
+      cy.visit(relative(`/case/${dealId}/facility/${facility._id}`));
+    });
+
+    after(() => {
+      cy.deleteDeals(dealId, ADMIN);
+    });
+
+    it('should display the default blank facility end date fields in the Details tab', () => {
+      facilityPage.facilityIsUsingFacilityEndDate().should('have.text', '-');
+      facilityPage.facilityFacilityEndDate().should('have.text', '-');
+      facilityPage.facilityBankReviewDate().should('not.exist');
+    });
+
+    it('should navigate to the bank review date page after selecting "No" to "Has the bank provided a facility end date"', () => {
+      amendmentsPage.navigateToIsUsingFacilityEndDatePage(true, false);
```

Comment:
Sorry to be a pain - I don't think I explained myself well enough. I think
```
      amendmentsPage.navigateToIsUsingFacilityEndDatePage(true, false);
```
is still confusing, but
```
navigateToIsUsingFacilityEndDatePage({ startNewAmendment: true boolean }) 
```
Would be more clear?

Reply:
updated